include/swift/AST/FineGrainedDependencies.h

```diff
@@ -346,6 +347,7 @@ class BiIndexedTwoStageMap {
 /// Write out the .swiftdeps file for a frontend compilation of a primary file.
 bool emitReferenceDependencies(DiagnosticEngine &diags, SourceFile *SF,
                                const DependencyTracker &depTracker,
+                               const ReferencedNameTracker *nameTracker,
```

Comment:
I'm guessing that the ReferencedNameTracker is no longer in the SourceFile, is that why it is now passed in separately? I'm wondering why it was moved out??

Reply:
The referenced name tracker is still on the SourceFile, but in order to support the fallback-to-manual path I've added it as a parameter.